### PR TITLE
Add 10-10EZ American Indian Question E2E spec

### PR DESCRIPTION
**Pull request by tlei123**
## Description

1. Add Cypress E2E-spec for 10-10EZ (HCA) American Indian Question.
2. Move all HCA E2E-specs, fixtures, mocks, schemas to new e2e folder.

## Original issue(s)
\[n/a\]

## Testing done
- Ran new HCA spec locally, and [successfully posted results to TestRail](https://dsvavsp.testrail.io/index.php?/runs/view/2818) \[see also screenshot below\].
- Ran other (moved) HCA specs locally, and all Passed.
- All E2E specs [passed in CI GHA-workflow](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/1680330036).

## Screenshot
**TestRail test-results for new HCA-AIQ spec**:

## Acceptance criteria
- [ ] Code reviewers approved changes

## Definition of done
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/hca/tests/e2e/application-status.cypress.spec.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-import mockUser from './fixtures/mocks/mockUser';
+/* eslint-disable va/axe-check-required */
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/hca/tests/e2e/application-status.cypress.spec.js:1]: This spec calls Platform's custom [testStatus](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/cypress/support/commands/testStatus.js) Cypress-command which already has `cy.axeCheck()` included.

File: src/applications/hca/tests/e2e/fixtures/mockUserAiq.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
+const VA_FORM_IDS = require('platform/forms/constants').VA_FORM_IDS;
+
+/* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/hca/tests/e2e/fixtures/mockUserAiq.js:3]: Disable's required for camel-cased keys in mockUser data.

File: src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    cy.intercept('POST', '/v0/health_care_applications', {
+      statusCode: 200,
+      body: {
+        formSubmissionId: '123fake-submission-id-567',
+        timestamp: moment().format('YYYY-MM-DD'),
+      },
+    }).as('mockSubmit');
+  });
+
+  /* eslint-disable va/axe-check-required */
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js:45]: AXE-check is in the last test.  All 4 tests check the same page, so no point in AXE-checking 4 times.

File: src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+      selector: '.additional-info-title',
+    })
+      .scrollIntoView()
+      .click()
+      .then(ele => {
+        cy.wrap(ele)
+          .parent()
+          .invoke('attr', 'aria-controls')
+          .then(ariaCtrlsId => {
+            // eslint-disable-next-line prefer-template
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js:157]: Will fix w/ next push.

File: src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js:177]: Will fix w/ next push.

File: src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js]: Fixed.

File: src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js]: Fixed.

**Review by timwright12 — APPROVED** [review]

**Review by lzapson — APPROVED** [review]
looks good to me

**Pull request merged**